spec/lib/pg_search/multisearchable_spec.rb

```diff
@@ -69,8 +73,13 @@
         context "when the document is missing" do
           before { record.pg_search_document = nil }
 
-          describe "saving the record" do
+          describe "saving the record without changing any in it" do
+            it "should not create a PgSearch::Document record" do
```

Comment:
Its all set we're back to normal!
ᐧ


Roger S. Wood
Vice President of Sales & Marketing
(978) 808-3675
roger@listenloop.com


On Wed, Sep 9, 2015 at 1:03 AM, Andrew Marshall <notifications@github.com>
wrote:

> In spec/lib/pg_search/multisearchable_spec.rb
> <https://github.com/Casecommons/pg_search/pull/218#discussion_r39007629>:
>
> > @@ -69,8 +73,13 @@
> >          context "when the document is missing" do
> >            before { record.pg_search_document = nil }
> >
> > -          describe "saving the record" do
> > +          describe "saving the record without changing any in it" do
> > +            it "should not create a PgSearch::Document record" do
>
> This is actually wrong, anyway, I think. If the PgSearch document is
> missing, we should still create it even if no data changed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Casecommons/pg_search/pull/218/files#r39007629>.
>